Packs/XSOARmirroring/Integrations/XSOARmirroring/XSOARmirroring.py

```diff
@@ -279,6 +283,9 @@ def fetch_incidents(client: Client, max_results: int, last_run: Dict[str, Union[
     """
 
     last_fetch = last_run.get('last_fetch')
+    # Drop playbookId field if configured.
+    if drop_playbookid and 'playbookId' in FIELDS_TO_COPY_FROM_REMOTE_INCIDENT:
+        FIELDS_TO_COPY_FROM_REMOTE_INCIDENT.remove('playbookId')
```

Comment:
Let's pass the parameter directly into `fetch_incidents`, and have it not copy this specific field, instead of changing the list of fields (which may cause trouble).